### PR TITLE
A large portion of the Atari Gran Trak 10 schematics (1974) implemented on MAME using the netlist subsystem.

### PR DESCRIPTION
**Pull request by felipesanches**
This is not the complete schematics, but I'd like to make it public because I would appreciate talking to other devs about it.

### CODE REVIEW

**Comment by felipesanches** [comment]
There's some video output being generated by this circuit and already being rendered by MAME, but the image is not yet stable.

Also, I am not sure I did correctly implement the 2 new nld_devices included in this pull request. I'd appreciate some code review.

**Comment by galibert** [comment]
TBH, we don't currently handle interlaced video.

  OG.

File: src/mame/drivers/atarittl.cpp [hunk 1]
```diff
@@ -192,6 +197,23 @@ class stuntcyc_state : public driver_device
...
+	{
+	}
+
+	void gtrak10(machine_config &config);
+
+protected:
+        // driver_device overrides
+        virtual void machine_start() override { };
+        virtual void machine_reset() override { };
+        virtual void video_start() override  { };
```
Comment (cuavas) [inline src/mame/drivers/atarittl.cpp]: There's no point overriding these if you're not doing anything.  Also you have superfluous semicolons.

File: src/mame/drivers/atarittl.cpp [hunk 2]
```diff
@@ -646,23 +711,23 @@ ROM_END
...
-GAME(1975,  steeplec,  0,        atarikee, 0, atarikee_state, empty_init, ROT0,  "Atari",        "Steeplechase [TTL]",     MACHINE_IS_SKELETON)
-GAME(1976,  stuntcyc,  0,        stuntcyc, 0, stuntcyc_state, empty_init, ROT0,  "Atari",        "Stunt Cycle [TTL]",      MACHINE_IS_SKELETON)
-GAME(1974,  tank,      0,        atarikee, 0, atarikee_state, empty_init, ROT0,  "Atari/Kee",    "Tank/Tank Cocktail [TTL]",     MACHINE_IS_SKELETON)
-GAME(1975,  tankii,    0,        atarikee, 0, atarikee_state, empty_init, ROT0,  "Atari/Kee",    "Tank II [TTL]",          MACHINE_IS_SKELETON)
+GAME(1975,  antiairc,  0,        atarikee,       0, atarikee_state, empty_init, ROT0,  "Atari",        "Anti-Aircraft [TTL]",    MACHINE_IS_SKELETON)
+GAME(1975,  crashnsc,  0,        atarikee,       0, atarikee_state, empty_init, ROT0,  "Atari",        "Crash 'n Score/Stock Car [TTL]",   MACHINE_IS_SKELETON)
+GAME(1974,  gtrak10,   0,         gtrak10, gtrak10,  gtrak10_state, empty_init, ROT0,  "Atari/Kee",    "Gran Trak 10/Trak 10/Formula K [TTL]",     MACHINE_NO_SOUND)
```
Comment (cuavas) [inline src/mame/drivers/atarittl.cpp]: Is the game playable as originally intended?  If not, the `MACHINE_NOT_WORKING` flag should be retained.

File: src/mame/machine/nl_gtrak10.cpp [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+	// --------------- Hack ----------------------
+	// This is a signal only used for debugging.
+	// It generates a pattern of squares on the screen.
+	// CAR1VIDEO = not(16H & 16V | VBLANK | HBLANK)
+
+	#ifdef SIGNAL_RT_HACK
+	#ifdef SIGNAL_RT_HACK_XOR
+	TTL_7486_XOR(16H_xor_16V, 16H, 16V)
+	TTL_7402_NOR(nor, 16H_xor_16V, BLANK)
+	#else
```
Comment (cuavas) [inline src/mame/machine/nl_gtrak10.cpp]: Please outdent preprocessor directives to the margin - they aren't affected by scope blocks.

**Comment by felipesanches** [comment]
I've just force-pushed an amended commit addressing all points that @cuavas brought up in the code review.

**Comment by felipesanches** [comment]
> Why are you crediting Ryan Holtz in several of your new files rather than yourself?

I try to keep whatever name was in files I copy and adapt. But maybe you're right. Some of the files may be almost entirely new code. I'll look carefully and update that line based on who actually worked on each piece of code. Thanks for the reminder.

**Comment by felipesanches** [comment]
ok. I have just force-pushed an amended commit addressing Haze's comments. Thanks, @DavidHaywood !

**Comment by felipesanches** [comment]
Thanks!

**Pull request merged**